support/java/Runtime.java

```diff
@@ -81,6 +78,7 @@ protected List getFieldOrder() {
         }
 
         public void mono_embeddinator_set_assembly_path(String path);
+        public void mono_set_dirs(String assemblyDir, String configDir);
```

Comment:
Going by what @jonpryor mentioned, we should not call Mono functions directly, especially since this is looking them up on the current library, and those symbols might not be exported in the future if we stop linking with monosgen directly on the command line.

Let's do this in the Embeddinator C runtime support code, you can `#if defined(__ANDROID__)` if need be.